python/cudf/cudf/dataframe/column.py

```diff
@@ -198,6 +199,9 @@ def __init__(self, data, mask=None, null_count=None, name=None):
 
         self._update_null_count(null_count)
 
+    def __contains__(self, item):
+        return item in columnops.as_column(self._data)
```

Comment:
This should instead raise a NotImplementedError if the implementation is going to live in downstream Column classes